modules/rostests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 
 start_module_group(rostests)
 
+add_definitions(-DHAVE_APITEST)
```

Comment:
Maybe something like `IS_REACTOS_APITEST` ?

Reply:
`HAVE_APITEST` is simple and familiar to CMake / `autoconf` culture.